rlmolecule/alphazero/alphazero_problem.py

```diff
@@ -2,33 +2,33 @@
 import os
 import random
 from abc import abstractmethod
-from typing import Optional
+from typing import List, Optional
 
+import numpy as np
 import sqlalchemy
 from sqlalchemy import exc
 
 from rlmolecule.alphazero.alphazero_vertex import AlphaZeroVertex
 from rlmolecule.mcts.mcts_problem import MCTSProblem
-from rlmolecule.sql import Base, Session
-from rlmolecule.sql.query import get_existing_reward
-from rlmolecule.sql.tables import GameStore, RewardStore
+from rlmolecule.sql import Base, Session, digest, load_numpy_dict, serialize_ordered_numpy_dict
+from rlmolecule.sql.tables import GameStore, RewardStore, StateStore
 from rlmolecule.tree_search.graph_search_state import GraphSearchState
-from rlmolecule.tree_search.reward import Reward
+from rlmolecule.tree_search.reward import Reward, RewardFactory
 
 logger = logging.getLogger(__name__)
 
 
 class AlphaZeroProblem(MCTSProblem):
 
-    def __init__(self,
+    def __init__(self, *,
```

Comment:
what does this do?

Reply:
nm, ok

Reply:
Yeah, I thought it was bad practice to provide a default rewardfactory, that's just too critical a parameter to get overlooked.
It was easier to deal with the inheritance with keyword only arguments